pkg/exporter/openapi3_test.go

```diff
@@ -65,8 +65,8 @@ func TestExport(t *testing.T) {
 	var app2 = syslwrapper.MakeApp("app2", []*sysl.Param{}, map[string]*sysl.Type{"request": type2})
 	var mod = &sysl.Module{
 		Apps: map[string]*sysl.Application{
-			"app1": &app1,
-			"app2": &app2,
+			"app1": app1,
```

Comment:
Ahh, right this is what you were referring to! 

This is because updating the sysl.pb.go changes the proto type definitions, which means the linter gives warnings on the way the application is passed around (by pointer rather than reference)

Should've included this in the commit message!